lingui.config.ts

```diff
@@ -1,6 +1,6 @@
 /** @type {import('@lingui/conf').LinguiConfig} */
 module.exports = {
-  locales: ['en', 'es', 'ja', 'fr', 'pt'],
+  locales: ['en', 'es', 'ja', 'fr', 'pt', 'zh-CN', 'ru', 'de', 'uk', 'sv-SE', 'fi', 'nl', 'el', 'it', 'pl'],
```

Comment:
I think we don't need the subset of languages right now
We can use the main set for now
I tested it on my repo and it was correct.

fix: 

zh-CN  ---> zh
sv-SE  ----> sv

Subsequently, the names of all components and files should be changed.

Reply:
fixed